### DISCUSSION TITLE
Floating window showing generation information

### DISCUSSION THREAD

**Discussion by PassWithoutTrace** [zanllp/infinite-image-browsing#333] [Q&A] [answered]
When browsing a single image, I used to have a floating window that showed the generational information. There were two icons that indicated that these windows could be dragged around to be placed elsewhere on the screen. I no longer can find the floating window. How can I get it back? I tried moving my infinite-image-browsing folder out of Extensions and re-installing it in the hopes of clearing preferences but that did not help. I'm aware that I can click in the top-right of a thumbnail to get this information, but the floating window was very convenient as I scrubbed through a collection at full-size. Please help.

**Comment by zanllp** [comment]
You can take a look at #295. 

It seems like I missed the check when opening, which caused this issue. Otherwise, after you upgraded to the latest version, this problem shouldn't have occurred. For now, you can temporarily address the issue based on #295.

**Comment by zanllp** [comment, marked as answer]
You can also resolve this issue by updating to the latest version, as I have added a check for it.
